thunder/torch/experimental/dtensor_utils.py

```diff
@@ -0,0 +1,103 @@
+from collections.abc import Sequence
+from typing import Any
+
+import torch
+from torch.distributed.tensor import DTensor
+
+from thunder.core.pytree import tree_map
+from thunder.core.proxies import TensorProxy, NumberProxy
+from thunder.core.devices import to_torch_device
+from thunder.core.dtypes import to_torch_dtype
+from thunder.core.pytree import tree_map
+from thunder.core.trace import TraceCtx
+from thunder.torch.experimental.dtensor_proxy import DTensorProxy
+from thunder.core.prims import PrimIDs
+from thunder.core.symbol import Symbol
+from thunder.core.trace import from_trace
+
+from torch._subclasses.fake_tensor import FakeTensorMode
+
+
+def run_with_fake_tensor(torch_op, *args, **kwargs):
```

Comment:
I've updated the implementation of this function to not use `aot_function` because the FX graph output was unused in this pull request.